lesson_37/first-react-app/src/components/Products.jsx

```diff
@@ -0,0 +1,10 @@
+import Product from "./Product"
+import "../css/products.css"
```

Comment:
in React convention better to create a folder Products and move there .jsx and .css file together